services/core-api/app/api/mines/permits/permit_conditions/models/permit_conditions.py

```diff
@@ -46,6 +47,15 @@ class _ModelSchema(Base._ModelSchema):
         order_by='asc(PermitConditions.display_order)',
         backref=backref('parent', remote_side=[permit_condition_id]))
 
+    update_user = db.Column(
```

Comment:
I don't believe we want to expose the Audit columns to the FE. 

Can you do something similar to `last_updated_by` and `last_updated_date` on `now_application` ?

Reply:
@nayr974 @JustinMacaulay I might be wrong about the audit columns, can you guys confirm?

Reply:
> @nayr974 @JustinMacaulay I might be wrong about the audit columns, can you guys confirm?

That's correct. `update_user`, `update_timestamp`, `create_user`, and `create_timestamp` fields are to never be exposed